Source/Core/DolphinWX/Android/ButtonManager.h

```diff
@@ -35,137 +35,147 @@ namespace ButtonManager
 		TRIGGER_L = 20,
 		TRIGGER_R = 21,
 		// Wiimote
-		WIIMOTE_BUTTON_A = 22,
-		WIIMOTE_BUTTON_B = 23,
-		WIIMOTE_BUTTON_MINUS = 24,
-		WIIMOTE_BUTTON_PLUS = 25,
-		WIIMOTE_BUTTON_HOME = 26,
-		WIIMOTE_BUTTON_1 = 27,
-		WIIMOTE_BUTTON_2 = 28,
-		WIIMOTE_UP = 29,
-		WIIMOTE_DOWN = 30,
-		WIIMOTE_LEFT = 31,
-		WIIMOTE_RIGHT = 32,
-		WIIMOTE_IR_UP = 33,
-		WIIMOTE_IR_DOWN = 34,
-		WIIMOTE_IR_LEFT = 35,
-		WIIMOTE_IR_RIGHT = 36,
-		WIIMOTE_IR_FORWARD = 37,
-		WIIMOTE_IR_BACKWARD = 38,
-		WIIMOTE_IR_HIDE = 39,
-		WIIMOTE_SWING_UP = 40,
-		WIIMOTE_SWING_DOWN = 41,
-		WIIMOTE_SWING_LEFT = 42,
-		WIIMOTE_SWING_RIGHT = 43,
-		WIIMOTE_SWING_FORWARD = 44,
-		WIIMOTE_SWING_BACKWARD = 45,
-		WIIMOTE_TILT_FORWARD = 46,
-		WIIMOTE_TILT_BACKWARD = 47,
-		WIIMOTE_TILT_LEFT = 48,
-		WIIMOTE_TILT_RIGHT = 49,
-		WIIMOTE_SHAKE_X = 51,
-		WIIMOTE_SHAKE_Y = 52,
-		WIIMOTE_SHAKE_Z = 53,
+		WIIMOTE_BUTTON_A = 100,
+		WIIMOTE_BUTTON_B = 101,
+		WIIMOTE_BUTTON_MINUS = 102,
+		WIIMOTE_BUTTON_PLUS = 103,
+		WIIMOTE_BUTTON_HOME = 104,
+		WIIMOTE_BUTTON_1 = 105,
+		WIIMOTE_BUTTON_2 = 106,
+		WIIMOTE_UP = 107,
+		WIIMOTE_DOWN = 108,
+		WIIMOTE_LEFT = 109,
+		WIIMOTE_RIGHT = 110,
+		WIIMOTE_IR = 111, /* To Be Used on Java Side */
+		WIIMOTE_IR_UP = 112,
+		WIIMOTE_IR_DOWN = 113,
+		WIIMOTE_IR_LEFT = 114,
+		WIIMOTE_IR_RIGHT = 115,
+		WIIMOTE_IR_FORWARD = 116,
+		WIIMOTE_IR_BACKWARD = 117,
+		WIIMOTE_IR_HIDE = 118,
+		WIIMOTE_SWING = 119, /* To Be Used on Java Side */
+		WIIMOTE_SWING_UP = 120,
+		WIIMOTE_SWING_DOWN = 121,
+		WIIMOTE_SWING_LEFT = 122,
+		WIIMOTE_SWING_RIGHT = 123,
+		WIIMOTE_SWING_FORWARD = 124,
+		WIIMOTE_SWING_BACKWARD = 125,
+		WIIMOTE_TILT = 126, /* To Be Used on Java Side */
+		WIIMOTE_TILT_FORWARD = 127,
+		WIIMOTE_TILT_BACKWARD = 128,
+		WIIMOTE_TILT_LEFT = 129,
+		WIIMOTE_TILT_RIGHT = 130,
+		WIIMOTE_TILT_MODIFIER = 131,
+		WIIMOTE_SHAKE_X = 132,
+		WIIMOTE_SHAKE_Y = 133,
+		WIIMOTE_SHAKE_Z = 134,
 		//Nunchuk
-		NUNCHUK_BUTTON_C = 54,
-		NUNCHUK_BUTTON_Z = 55,
-		NUNCHUK_STICK = 56,
-		NUNCHUK_STICK_UP = 57,
-		NUNCHUK_STICK_DOWN = 58,
-		NUNCHUK_STICK_LEFT = 59,
-		NUNCHUK_STICK_RIGHT = 60,
-		NUNCHUK_SWING_UP = 61,
-		NUNCHUK_SWING_DOWN = 62,
-		NUNCHUK_SWING_LEFT = 63,
-		NUNCHUK_SWING_RIGHT = 64,
-		NUNCHUK_SWING_FORWARD = 65,
-		NUNCHUK_SWING_BACKWARD = 66,
-		NUNCHUK_TILT_FORWARD = 67,
-		NUNCHUK_TILT_BACKWARD = 68,
-		NUNCHUK_TILT_LEFT = 69,
-		NUNCHUK_TILT_RIGHT = 70,
-		NUNCHUK_SHAKE_X = 72,
-		NUNCHUK_SHAKE_Y = 73,
-		NUNCHUK_SHAKE_Z = 74,
+		NUNCHUK_BUTTON_C = 200,
+		NUNCHUK_BUTTON_Z = 201,
+		NUNCHUK_STICK = 202, /* Used on Java Side */
```

Comment:
As do I, normally, but when someone else has begun with a different style, I tend to follow that (see line 25 and line 30 in this file).